sympy/polys/polyutils.py

```diff
@@ -420,6 +437,28 @@ def _dict_reorder(rep, gens, new_gens):
 
     return map(tuple, new_monoms), coeffs
 
+def has_imaginary(exprs):
+    """Checks whether the numerator, denominator of an expression has Imaginary Unit"""
+    has_imag_in_num_denom = [False, False]
```

Comment:
I would suggest making this `in_num, in_den = [False]*2` and return `in_num, in_den` at the end rather than use an index which will force someone to look back to make sure that the order is num/den (though I see you use the name to give that hint). Still...I would prefer to see direct rather than indirect referencing.